### ISSUE TITLE
Widget image does not have ALT attribute

### ISSUE THREAD

**Issue by inghamemerson** [lytics/pathforajs#637]
Widget images do not set the `alt` attribute when [rendered](https://github.com/lytics/pathforajs/blob/1be021a008c066b841b4b50c3eb0ea622d9f2d0c/src/rollup/widgets/construct-widget-layout.js#L189-L200). Ideally this would default to the "null" value so that AT can safely ignore the image [per WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/H67.html).

From what I can find, it doesn't appear that an alt text attribute is stored in the product and using the headline property would not be descriptive.

**Comment by ashyablok** [comment]
This was merged and deployed
